art/estimators/estimator.py

```diff
@@ -512,6 +512,20 @@ def set_learning_phase(self, train: bool) -> None:
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def loss(self, x: np.ndarray, y: np.ndarray, **kwargs) -> np.ndarray:
+        """
+        Compute the loss of the neural network for samples `x`.
```

Comment:
Estimators don't have to be neural networks.
```suggestion
        Compute the loss of the model for samples `x`.
```